_posts/2020-09-14-Introduction-to-API-Management.md

```diff
@@ -0,0 +1,284 @@
+It's not my intention to go to much in dept of Apiman or the settings, but a few base principles I want to highlight to help you understand how easy it was to try it out and play with it.  
+The first very important part in Apiman was that everything starts with an Organization.  
+2ndly you need to setup a plan, which is basically a collection of policies that will be applied to requests made to Services being access through it.  
+As last you need to setup a Service. 
+A service contract is simply a link between an application and a service through a plan offered by that service. 
+When a service contract is created, the system generates a unique API key specific to that contract. 
+All requests made to the service through the API Gateway must include this API key.  
+
+Apiman start Screen.  
+<img src="/img/2020-09-14-Introduction-to-API-Management/APIMAN_Indexpage.PNG" alt="APIMAN start screen" width="600" height="375" class="image fit" style="vertical-align:middle;margin-left:2%" />
+
+Setting up an organization (OrgHome) and creating services.
+
+<img src="/img/2020-09-14-Introduction-to-API-Management/APIMAN_Organization-API.PNG" alt="APIMAN Organization and Services" width="800" height="300" class="image fit" style="vertical-align:middle;margin-left:2%" />
+
+In this example you can see a Customer API which fetches Customer information from a Spring Boot application running in IntelliJ.  
+Every service can be marked public, so that everyone can access it, or you can for example secure it by creating a Client App and generate a required key via a plan.   
+In next screen you can see that I created a 'SearchCustomers' client app, where the client needs to use the API-Key in the request to get the response back from our service.
+Our service (Spring Boot application) didn't implement any security or authority for this.
+Apiman will take the responsibility of implementing this.
+Only if the client uses the right endpoint with the correct API-key, Apiman will retrieve the requested data from our backend service and send the response back to the client.
+ 
+<img src="/img/2020-09-14-Introduction-to-API-Management/APIMAN_ClientApps-APIKey.PNG" alt="APIMAN Client apps and API-Key" width="800" height="400" class="image fit" style="vertical-align:middle;margin-left:2%" />
+
+### Automation
+
+As last I wanted to be sure that we can use the tool in an automated way, so I tried to use the Apiman REST API to create new organizations, plans and services.
+I had some struggles to make it work, but eventually I was able to create everything I wanted.
+Via the REST API we have the possibility to check, add or remove anything you want.
+The build in Apiman-UI is also build on these REST API, so anything you can do in the application should be possible to execute via the API service.
+More on this can be find in the <a href="http://www.apiman.io/latest/api-manager-restdocs.html" target="_blank" rel="noopener noreferrer">API-manager-restdocs</a>.

```

Comment:
```suggestion
More on this can be found in the <a href="http://www.apiman.io/latest/api-manager-restdocs.html" target="_blank" rel="noopener noreferrer">API-manager-restdocs</a>.

```